app/assets/stylesheets/sidebar-data.scss

```diff
@@ -6,6 +6,13 @@
     color: $dark-green;
     font-family: $monospace;
     text-transform: lowercase;
+    margin: 8px 0;
+    padding: 8px 0;
+    font-size: 0.9em;
+  }
+  p {
+    margin: 8px 0;
+    padding: 0;
```

Comment:
thanks for taking care of that!

Reply:
Thanks for taking care of the spacing issues for the overall reader stats.